docs/start_guide.md

```diff
@@ -0,0 +1,232 @@
+
+For this example, we will be using the [helloapache](https://github.com/projectatomic/nulecule-library/tree/master/helloapache) example from the [nulecule-library](https://github.com/projectatomic/nulecule-library) repo.
+
+In order to follow along, fetch the container and `cd` into the directory:
+```sh
+atomicapp fetch --destination helloapache projectatomic/helloapache
+cd helloapache
+```
+
+### ~/helloapache/Dockerfile
+Atomic App itself is packaged as a container. End-users typically do not install the software from source, instead using the `atomicapp` container as the `FROM` line in a Dockerfile and packaging your application on top. For example:
+
+```Dockerfile
+FROM projectatomic/atomicapp
+
+MAINTAINER Your Name <you@example.com>
+
+ADD /Nulecule /Dockerfile README.md /application-entity/
+ADD /artifacts /application-entity/artifacts
+```
+
+Within `helloapache` we specify a bit more within our labels:
+```Dockerfile
+FROM projectatomic/atomicapp:0.4.2
+
+MAINTAINER Red Hat, Inc. <container-tools@redhat.com>
+
+LABEL io.projectatomic.nulecule.providers="kubernetes,docker,marathon" \
+      io.projectatomic.nulecule.specversion="0.0.2"
+
+ADD /Nulecule /Dockerfile README.md /application-entity/
+ADD /artifacts /application-entity/artifacts
+```
+
+__Optionally__ indicating what providers are included within our container:
+```Dockerfile
+LABEL io.projectatomic.nulecule.providers="kubernetes,docker" \
+      io.projectatomic.nulecule.specversion="0.0.2"
+```
+
+### ~/helloapache/Nulecule
+
+This is the `Nulecule` file for Atomic App. The `Nulecule` file is composed of graph and metadata in order to link one or more containers for your application.
+```yaml
+---
+specversion: 0.0.2
+id: helloapache-app
+
+metadata:
+  name: Hello Apache App
+  appversion: 0.0.1
+  description: Atomic app for deploying a really basic Apache HTTP server
+
+graph:
+  - name: helloapache-app
+
+    params:
+      - name: image
+        description: The webserver image
+        default: centos/httpd
+      - name: hostport
+        description: The host TCP port as the external endpoint
+        default: 80
+
+    artifacts:
+      docker:
+        - file://artifacts/docker/hello-apache-pod_run
+      kubernetes:
+        - file://artifacts/kubernetes/hello-apache-pod.json
+      marathon:
+        - file://artifacts/marathon/helloapache.json
+```
+
+##### Spec and id information 
+```yaml
+---
+specversion: 0.0.2
+id: helloapache-app
+```
+
+Here we indicate the specversion of our Atomic App (similar to a `v1` or `v2` of an API) as well as our ID.
+
+##### Metadata
+```yaml
+metadata:
+  name: Hello Apache App
+  appversion: 0.0.1
+  description: Atomic app for deploying a really basic Apache HTTP server
+```
+
+An __optional__ requirement, but this is the metadata associated with your packaged container. A good description as well as a version number will help indicate other users what your Nuleculized container is all about.
```

Comment:
We probably shouldn't use the words "optional requirement". We should probably just say "Optionally you can add a metadata section...."